function-server/main.py

```diff
@@ -5,18 +5,23 @@
 import signal
 import sys
 import traceback
+from concurrent.futures import ThreadPoolExecutor
+from concurrent.futures._base import TimeoutError
```

Comment:
TimeoutError seems to be available in futures package:
```
>>> import concurrent.futures
>>> concurrent.futures.TimeoutError
<class 'concurrent.futures._base.TimeoutError'>
```

Reply:
weird, is this something special about python packages that I'm not aware of? The error is shown as `concurrent.futures._base.TimeoutError` in the traceback, but it still works if I use `concurrent.futures.TimeoutError`

Reply:
It's reimported in the parent package: 
https://github.com/python/cpython/blob/3.6/Lib/concurrent/futures/__init__.py